README.md

```diff
@@ -102,6 +102,30 @@ use OmniAuth::Builder do
 end
 ```
 
+#### Upgrading to 1.0+
+
+Versions before `1.0` allowed you to pass `:scope` option as a block, just as today.
+However, that block received a `Rack::Request` instance, rather than the `Rack` `env`.
+If you used that given `Rack::Request` in your `:scope` block you can update your code create a new instance, from the `env`.
```

Comment:
s/you can update your code create a new instance/you can update your code and create a new instance/

Reply:
Good catch. Thank you!